apps/common-app/src/examples/AnimatedKeyboardExample.tsx

```diff
@@ -1,88 +1,168 @@
+import React from 'react';
 import Animated, {
-  KeyboardState,
   useAnimatedKeyboard,
   useAnimatedStyle,
+  useAnimatedProps,
 } from 'react-native-reanimated';
 import {
-  Button,
   Keyboard,
   ScrollView,
   StyleSheet,
   TextInput,
+  TouchableOpacity,
   View,
+  Text,
 } from 'react-native';
 
-import React from 'react';
+Animated.addWhitelistedNativeProps({ text: true });
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
 
-const BOX_SIZE = 50;
+const KeyboardStateToName = {
+  0: 'UNKNOWN',
+  1: 'OPENING',
+  2: 'OPEN',
+  3: 'CLOSING',
+  4: 'CLOSED',
+};
 
-function NestedView() {
-  useAnimatedKeyboard();
-  return <View style={styles.nestedView} />;
+function getRandomColor() {
+  const letters = '0123456789ABCDEF';
+  let color = '#';
+  for (let i = 0; i < 6; i++) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
+
+function RandomView({ number }: { number: number }) {
```

Comment:
Using `number` as a variable name is a bad idea.

```suggestion
function RandomView({ id }: { id: number }) {
```